18F/identity-idp
# LG-6625: Hide IPP troubleshooting option for failures

**Why**: So that the option to proof in person is only extended to those who would be successful in proofing.

**Testing Instructions:**

1. Navigate to http://localhost:3000
2. Sign in
3. Navigate to http://localhost:3000/verify
4. Complete proofing flow up to document capture
5. Use one of YAML document examples for field upload
6. Click "Submit"
7. Observe that "Verify your ID in person at a Post Office" only appears for non-"Failed" result

**Document Examples:**

`attention.yml`:

```yml
failed_alerts:
  - name: Document Classification
    result: Attention
```

`failed.yml`:

```yml
doc_auth_result: Failed
```

**Screenshots:**

Failure Type|Screenshot
---|---
Attention|![image](https://user-images.githubusercontent.com/1779930/181035188-c4d1336f-09f6-4cd7-b23c-2af8a7432c77.png)
Failure|![image](https://user-images.githubusercontent.com/1779930/181035238-f1a7d64f-addc-46d3-96f8-166a527bf1eb.png)
